### PR TITLE
CRM-20947, fixed notice error on Activity tab

### PR DESCRIPTION
**Pull request by pradpnayak**
Overview
----------------------------------------
This PR fixes notice error when trying to save activity and viewing list of activity in Activity tab.

Before
----------------------------------------
noticeerror

After
----------------------------------------
No error.

Technical Details
----------------------------------------
This PR fixes the notice error of using deprecated function. I have replaced it with CRM_Core_PseudoConstant::getKey() method to retrieve value of option values.

---

 * [CRM-20947: Remove Deprecation Notice for Option group](https://issues.civicrm.org/jira/browse/CRM-20947)

### CODE REVIEW

File: CRM/Activity/BAO/Activity.php [hunk 1]
```diff
@@ -2373,7 +2373,10 @@ public static function createFollowupActivity($activityId, $params) {
     $followupParams = array();
     $followupParams['parent_id'] = $activityId;
     $followupParams['source_contact_id'] = CRM_Core_Session::getLoggedInContactID();
-    $followupParams['status_id'] = CRM_Core_OptionGroup::getValue('activity_status', 'Scheduled', 'name');
+    $followupParams['status_id'] = CRM_Core_PseudoConstant::getKey('CRM_Activity_BAO_Activity',
+      'activity_status',
```
Comment (eileenmcnaughton) [inline CRM/Activity/BAO/Activity.php]: shouldn't this be status_id

**Comment by eileenmcnaughton** [comment]
first one looks wrong, others look right

File: CRM/Activity/BAO/Activity.php (hunk 1 above)
Comment (pradpnayak) [inline CRM/Activity/BAO/Activity.php]: Oops sorry.. Not sure how i missed that. Pushed the change.

**Comment by eileenmcnaughton** [comment]
I thought that last one was wrong & it should be just status_id - however grepping says

CRM_Core_PseudoConstant::getKey('CRM_Activity_BAO_Activity', 'activity_status_id', 'Completed'),

These are simple changes but easy to make errors in - hopefully there won't be anything we both miss though!

**Comment by pradpnayak** [comment]
yes both works (status_id / activity_status_id). I feel the code takes as field name or fields unique name defined in 
https://github.com/civicrm/civicrm-core/blob/master/xml/schema/Activity/Activity.xml#L241

**Comment by eileenmcnaughton** [comment]
Ah cool!

**Pull request merged**